src/main/java/com/challenge/api/service/auth/AuthService.java

```diff
@@ -0,0 +1,98 @@
+package com.challenge.api.service.auth;
+
+import com.challenge.api.service.auth.request.KakaoLoginServiceRequest;
+import com.challenge.api.service.auth.request.KakaoSigninServiceRequest;
+import com.challenge.api.service.auth.response.LoginResponse;
+import com.challenge.api.service.auth.response.SocialInfoResponse;
+import com.challenge.api.validator.auth.AuthValidator;
+import com.challenge.domain.job.Job;
+import com.challenge.domain.job.JobRepository;
+import com.challenge.domain.member.LoginType;
+import com.challenge.domain.member.Member;
+import com.challenge.domain.member.MemberRepository;
+import com.challenge.domain.member.Year;
+import com.challenge.exception.ErrorCode;
+import com.challenge.exception.GlobalException;
+import com.challenge.utils.JwtUtil;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class AuthService {
+
+    private final AuthValidator authValidator;
+    private final KakaoApiService kakaoApiService;
+    private final MemberRepository memberRepository;
+    private final JobRepository jobRepository;
+    private final JwtUtil jwtUtil;
+
+    /**
+     * 카카오 로그인 메소드
+     * kakao access token을 이용해 사용자 정보 조회 jwt access token 발급
+     *
+     * @param request
+     * @return
+     */
+    @Transactional(readOnly = true)
+    public LoginResponse kakaoLogin(KakaoLoginServiceRequest request) {
```

Comment:
```suggestion
    @Transactional
    public LoginResponse kakaoLogin(KakaoLoginServiceRequest request) {
```
트랜잭션 설정을 실수하신것 같아요 ~

Reply:
카카오 로그인 메소드는 db 관련해서는 조회하는 로직만 있어서 readOnly로 했는데 false로 해야하나요..?!